src/helper/wrapper/playwrightWrappers.ts

```diff
@@ -25,6 +25,20 @@ export default class PlaywrightWrapper {
     await locator.fill(word);
   }
 
+  async clearAndType(locator: Locator, word: string) {
+    await locator.waitFor({
+      state: 'visible',
+      timeout: 30000
+    });
+
+    const valueLength = (await locator.inputValue()).length;
+    for (let i = 0; i < valueLength; i++) {
+      await locator.press('Backspace');
+    }
```

Comment:
Have you tried this: https://playwright.dev/docs/api/class-locator#locator-clear ?